documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "development Copy",
+	"values": [
+		{
+			"key": "email-from",
```

Comment:
Is this variable needed/used here? If so - for which endpoint in this simplified collection?